### ISSUE TITLE
Improve animation performance

### ISSUE THREAD

**Issue by amannn** [davidchin/react-input-range#39]
I've noticed that a change in value results into a change in `width` for the active track and a change of `left` for the slider container. Changing these properties causes unnecessary reflows and repaints (see here: [width](https://csstriggers.com/width) and [left](https://csstriggers.com/left)).

Would it make sense to change both of these to `transform` so only composite is triggered when the value is changed?

This would result into an improved animation performance and in general a more smooth feel as the elements can animate through "sub CSS pixels" on screens with a higher pixel density.

**Comment by oyeanuj** [comment]
@amannn Did you end up making these change on a fork/branch? or has this already been taken care of by @davidchin?

**Comment by amannn** [comment]
@oyeanuj No, I haven't got to implementing it. It seems like these properties are still [in use](https://github.com/davidchin/react-input-range/blob/master/src/js/input-range/track.jsx#L57).
